commerce-product-item-selector-web/src/main/java/com/liferay/commerce/product/item/selector/web/internal/display/context/BaseCPItemSelectorViewDisplayContext.java

```diff
@@ -194,11 +194,11 @@ protected RowChecker getRowChecker() {
 		return _rowChecker;
 	}
 
-	protected final CPRequestHelper cpRequestHelper;
 	protected final HttpServletRequest httpServletRequest;
 	protected final LiferayPortletRequest liferayPortletRequest;
 	protected final LiferayPortletResponse liferayPortletResponse;
 	protected final PortalPreferences portalPreferences;
+	protected final CPItemSelectorRequestHelper requestHelper;
```

Comment:
`CPItemSelectorRequestHelper requestHelper` should be `CPItemSelectorRequestHelper cpItemSelectorRequestHelper`